scripts/scripts.js

```diff
@@ -90,9 +90,65 @@ sampleRUM.observe = ((elements) => {
   });
 });
 
+sampleRUM.targetselector = (element) => {
+  let selector = element.tagName.toLowerCase();
+  // always include these attributes as they have valuable info
+  const distinctAttributes = ['src', 'href'];
+  // capture the actual click target instead of a child element
+  const terminateearly = (e) => e.tagName.toLowerCase() === 'a'
+    || e.tagName.toLowerCase() === 'button';
+  if (element === document.body || element === document.documentElement) {
+    return selector;
+  }
+  if (element.id) {
+    return `#${element.id}`;
+  }
+  if (element.className) {
+    selector = `${selector}.${Array.from(element.classList).join('.')}`;
+  }
+  selector = distinctAttributes.reduce((pv, attname) => {
+    if (element.getAttribute(attname)) {
+      return `${pv}[${attname}="${element.getAttribute(attname)}"]`;
```

Comment:
For links and images, we add the attributes under `distinctAttributes`, so that the click target can be seen at a glance, e.g.
```
main > div.section-wrapper.recommended-articles-small-container > div.recommended-articles-small-content-wrapper > div.recommended-articles--small-.recommended-articles.small.block > div.article-cards > a.article-card[href="/en/publish/2021/10/26/new-creative-cloud-releases-enable-creative-collaboration-drive-innovation-empower-creative-careers"]
```